maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -407,6 +408,19 @@ class IOSDriver(
         return element == null
     }
 
+    private fun tapContinueIfVisible() {
+        val continueFilter = Filters.textMatches("Continue".toRegex())
+        val continueElement = MaestroTimer.withTimeout(2000) {
+            continueFilter(contentDescriptor().aggregate()).firstOrNull()
+        }?.toUiElementOrNull()
+        if (continueElement != null) {
+            val center = continueElement.bounds.center()
+            if (center.y in heightPoints / 2..heightPoints) {
+                tap(center)
+            }
+        }
+    }
```

Comment:
Given the uniformity of apple and its ui keyboard until we have a custom keyboard in picture. Can we safely say that Continue above 63% region would be keyboard one?


<img width="1307" alt="Screenshot 2023-04-05 at 9 47 28 PM" src="https://user-images.githubusercontent.com/12881364/230142326-2cfcb383-c0d6-4875-8752-d518710c03a6.png">


cc: @berikv @artem888 let me know your thoughts here 🙌

Reply:
Lets be more specific:

Hierarchy on iPhone 14, which has 844 point height:
```
 {
                    "attributes" : {
                      "text" : "Speed up your typing by sliding your finger across the letters to compose a word.",
                      "resource-id" : "",
                      "bounds" : "[35,692][355,738]",
                      "enabled" : "true",
                      "focused" : "false",
                      "selected" : "false",
                      "checked" : "false"
                    },
                    "children" : [ ],
                    "enabled" : true,
                    "focused" : false,
                    "checked" : false,
                    "selected" : false
                  } ],
                  "enabled" : true,
                  "focused" : false,
                  "checked" : false,
                  "selected" : false
                } ],
                "enabled" : true,
                "focused" : false,
                "checked" : false,
                "selected" : false
              } ],
              "enabled" : true,
              "focused" : false,
              "checked" : false,
              "selected" : false
            }, {
              "attributes" : {
                "text" : "Continue",
                "resource-id" : "",
                "bounds" : "[83,789][307,810]",
                "enabled" : "true",
                "focused" : "false",
                "selected" : "false",
                "checked" : "false"
              },
              "children" : [ {
                "attributes" : {
                  "text" : "Continue",
                  "resource-id" : "",
                  "bounds" : "[157,788][233,810]",
                  "enabled" : "true",
                  "focused" : "false",
                  "selected" : "false",
                  "checked" : "false"
                },
                "children" : [ ],
                "enabled" : true,
                "focused" : false,
                "checked" : false,
                "selected" : false
              } ],
```

Options are:
* Text match the help text
* Match the view pattern (there should be the keyboard view attached to the screen bottom, with the button inside)
* Match the view position:
 * The position-bottom should be < 50 points from the screen bottom

Reply:
I'd suggest matching via this text for now "Speed up your typing by sliding your finger across the letters to compose a word." since it is going to be the fastest possible fix. 

Long-term `Match the view pattern` option is the best one, except `bounds` everything should be fixed and for `bounds` we could implement additional validation logic

Reply:
Note that we still have to match the bottom-most continue button, Safari for example also has a continue button on first launch